### PR TITLE
[Timeline] Group height mode

### PR DESCRIPTION
**Pull request by francescogc**
Hi everyone, long time lurker here, I absolutely love this project and since I have made several changes to the library over the past few months (for a project at work) I think it's time to say thank you with a couple of PRs.

Here's the first one: it adds a global parameter called `groupHeightMode` that can be overriden by a group and can have two values: 'auto' or 'fixed'. Basically it specifies how the height of a group gets calculated. If it's set to 'auto' the height is calculated based on the visibile items, like always. On the other hand, with the 'fixed' value the group keeps the same height even if there are no visible items in the timeline at that moment.

I think it's a nice feature to have, hope to see it in some future version :)

### CODE REVIEW

**Comment by mbroad** [comment]
I like it

File: docs/timeline/index.html [hunk 1]
```diff
@@ -469,6 +469,12 @@ <h3 id="groups">Groups</h3>
       <td>no</td>
       <td>Assuming the group has nested groups, this will set the initial state of the group - shown or collapsed. The <code>showNested</code> is defaulted to <code>true</code>.</td>
     </tr>
+    <tr>
+      <td>heightMode</td>
+      <td>String</td>
+      <td>no</td>
+      <td>This field is optional. If set this overrides the global <code>groupHeightMode</code> configuration option for this group.</td>
```
Comment (mojoaxel) [inline docs/timeline/index.html]: Can you please all the valid option ('auto' and 'fixed') to this description.

**Comment by mojoaxel** [comment]
@francescogc Also please update your feature branch to the current develop branch (`git merge origin/develop`)

File: docs/timeline/index.html (hunk 1 above)
Comment (francescogc) [reply docs/timeline/index.html]: Sure, done.

**Comment by francescogc** [comment]
@mojoaxel feature branch is now up to date but I can't scroll anymore in the timeline... Is it just for me? :\

**Comment by yotamberk** [comment]
I fixed it. My mistake in a previous PR.

**Pull request merged**